queries.sql

```diff
@@ -1,3 +1,28 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+/*Milestone 01 */
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
+SELECT *
+FROM animals
+WHERE date_of_birth BETWEEN '01-01-2016' AND '12-12-2019';
+SELECT name
+FROM animals
+WHERE neutered = True
+    AND escape_attempts < 3;
+SELECT date_of_birth
+FROM animals
+WHERE name NOT IN ('Agumon', 'Pikachu');
```

Comment:
- [ ] You should list the `Date of birth` of all animals named either "Agumon" or "Pikachu", kindly consider fixing your query for that e.g
```sql
SELECT date_of_birth 
FROM animals 
WHERE name = 'Pikachu' OR name = 'Agumon';
```